src/SFA.DAS.EAS.Web/Views/Transfers/TransferConfirmation.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿@using SFA.DAS.EmployerCommitments.Web.Extensions
+@model SFA.DAS.EmployerCommitments.Web.ViewModels.TransferConfirmationViewModel
+
+@{
+    ViewBag.Title = $"Transfer request {Model.TransferApprovalStatus}";
+    ViewBag.Section = "";
+    ViewBag.PageID = "transfer-confirmation";
+}
+
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <div class="success-summary">
+            <h1 class="heading-medium">@ViewBag.Title</h1>
+            <p>You have @Model.TransferApprovalStatus a transfer request from @Model.TransferReceiverName</p>
+        </div>
+    </div>
+</div>
+
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <form method="POST" action="@Url.Action("TransferConfirmation")" onsubmit="sfa.tagHelper.submitRadioForm('@ViewBag.Title')">
+
+            @Html.Partial("ValidationSummary", Html.ViewData.ModelState)
+
+            <fieldset role="radiogroup" aria-required="true">
+                <div class="form-error-group form-group @Html.AddClassIfPropertyInError("UrlAddress", "error")">
+                    <h2 class="heading-medium" for="">What next?</h2>
+                    <div>
+                        @Html.ValidationMessage("UrlAddress", "Please choose an option", new { id = "error-message-UrlAddress" })
+                    </div>
+                    <label class="block-label selection-button-radio" for="UrlAddress-Transfers">
+                        @Html.RadioButton("UrlAddress", Url.ExternalUrlAction("teams", "transfers"), new { id = "UrlAddress-Transfers", dataOptionName = "Yes", onclick = $"sfa.tagHelper.radioButtonClick('{ViewBag.Title}', 'Transfers')" })
```

Comment:
As before, I think Index of Transfers looks a better bet but worth confirming with MYA.

Reply:
Good spot. Will change